apps/fz_http/lib/fz_http_web/live/rule_live/rule_list_component.html.heex

```diff
@@ -111,18 +115,20 @@
                 <%= @users[rule.user_id] %>
               </td>
               <td class="has-text-left">
-                <%= port_type_display(rule.port_type) %>
+                <%= port_type_display(rule.protocol) %>
               </td>
               <td class="has-text-left">
-                <%= if rule.port_range != nil, do: rule.port_range %>
+                <%= if rule.port_range_start != nil, do: rule.port_range_start %>
```

Comment:
Is this conditional necessary? Wouldn't `Kernel.to_string(nil)` just output an empty string anyway?

Reply:
Haven't done much work on the UI, simply the bare minimum to get the Allow Rules running to easily test them along the gateway